docker-build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Builds the APK inside the Docker container
+
+set -o errexit
+set -o xtrace
+
+# Handle the release type
+case ${RELEASE} in
+    'production')
+        RELEASE_SUFFIX=""
+        NODE_ENV="production"
+        RFLAG="assembleRelease"
+    ;;
+    'unminified')
+        RELEASE_SUFFIX="unminified_"
```

Comment:
We don't use node so this section won't do anything, need to find a new way to specify debug as well.

Reply:
The `assembleDebug` does that, or do you mean specifying unminified?

Reply:
specifying unminified

Reply:
As discussed on Riot, might be OK to just not have this as this app seems to not have as many issues with minification. Worth research for the future though.